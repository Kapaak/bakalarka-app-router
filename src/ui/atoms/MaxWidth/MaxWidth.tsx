import { PropsWithChildren } from "react";

import { ClassName } from "@/domains";

interface MaxWidthProps extends PropsWithChildren {
  className?: ClassName;
}

export const MaxWidth = ({ children, className }: MaxWidthProps) => {
  return (
    <div className={`mx-auto max-w-screen-2xl px-8  md:px-14 ${className}`}>
      {children}
    </div>
  );
};
