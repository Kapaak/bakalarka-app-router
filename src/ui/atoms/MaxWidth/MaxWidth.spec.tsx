import { render } from "@testing-library/react";

import { MaxWidth } from "./MaxWidth";

describe("MaxWidth", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<MaxWidth />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
