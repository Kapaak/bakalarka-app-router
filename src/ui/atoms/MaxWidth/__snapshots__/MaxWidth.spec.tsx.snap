// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaxWidth should render successfully 1`] = `
<body>
  <div>
    <div
      class="mx-auto max-w-screen-2xl px-8  md:px-14 undefined"
    />
  </div>
</body>
`;
